src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/extender/ResetExtender.java

```diff
@@ -0,0 +1,29 @@
+package edu.greenblitz.tobyDetermined.commands.telescopicArm.extender;
+
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Elbow;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Extender;
+
+public class ResetExtender extends ExtenderCommand {
```

Comment:
Raz - why do we need to call the super methods?

Reply:
In order to have extender variable in the class (extender = extenderSubsystem.getInstance() in extenderCommand